readme.md

```diff
@@ -448,7 +448,7 @@ player.source = {
     ],
     poster: '/path/to/poster.jpg',
     previewThumbnails: {
-        src: '/path/to/thumbnails.vtt'
+        src: '/path/to/thumbnails.vtt',
```

Comment:
Seems like this is an error change, seems like there is always a comma even for the last element of objects